docs/hub/spaces-secrets-scanner.md

```diff
@@ -0,0 +1,23 @@
+# Spaces Secrets Scanner
+
+It is important to manage [your secrets (env variables) properly](./spaces-overview#managing-secrets). The most common way people expose their secrets to outside world is hard-coding their secrets in their `app.py` files directly, which makes it possible for a malicious user to utilise your secrets and services your secrets have access to. 
+
+For example, here is how example `app.py` described above might look like:
+
+```py
+import numpy as np
+import scipy as sp
+
+api_key = "sw-xyz1234567891213"
+
+def call_inference(prompt: str) -> str:
+    result = call_api(prompt, api_key)
+    return result
+```
+
+To prevent this isse, there is `Spaces Secrets Scanner` that scans for hard-coded secrets and opens a discussion (in case hard-coded secrets are found) about the exposed secrets & how to handle this problem.
```

Comment:
```suggestion
To prevent this issue, we run an automated bot (Spaces Secrets Scanner) that scans for hard-coded secrets and opens a discussion (in case hard-coded secrets are found) about the exposed secrets & how to handle this problem.
```